external/protobuf/README.md

```diff
@@ -3,7 +3,7 @@
 Protocol Buffers (a.k.a. `protobuf`) is Google's language-neutral, platform-neutral, extensible mechanism for serializing structured data. TizenRT features a port for `protobuf` version 3.5.1, with support for C++ language. 
 You can find details of `protobuf` at https://github.com/google/protobuf.
 
-The following sections detail the steps required to build `protobuf` on TizenRT, and to build and run TizenRT applications that use `protobuf`.
+The following sections detail the steps required to build `protobuf` on TizenRT, and to build and run TizenRT applications that use `protobuf`. **In case you have just recently installed `gRPC`, and wish to install `Protobuf` to work along with `gRPC`, please ignore this document, and refer to the installation instructions under [Installing Protobuf with gRPC](https://github.com/Samsung/TizenRT/blob/master/external/grpc/README.md#pre-requisites) instead.**
```

Comment:
When I used protobuf only before, but now, I want to use gRPC. Should I remove installed protobuf?
We had better guide more.

When I want to use gRPC, should I ignore this? or Can I install both (Is it just recommandation?)?

Reply:
This sentence has a different topic agaginst above line, ```The following ... use protobuf```. 
To say different topic, you had better change a line.